hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -1,11 +1,72 @@
 module HookContextExtension
+  # FIXME: remove when #23332 is released
```

Comment:
I'd like to keep this minimal. Didn't we have a separate PR for merging the hooks? Could you perhaps separate it into two commits at least?

Reply:
I can break out yet another PR for merging these extensions. Please try to suggest those types of changes as early as you can. This change blocks multiple other changes and is dragging on.

Reply:
I don't require a separate PR, but splitting changes into logical separate commits gives a cleaner git log. Personally I often use that. So one commit to move everything from Katello and another to add the additional method.